docs/Notes.puml

```diff
@@ -0,0 +1,172 @@
+//@@author WilsonLee2000
+
+@startuml
+title Sequence Diagram for Notes Functionalities
+'https://plantuml.com/sequence-diagram
+
+actor User
+
+User -> BadMaths : main()
+activate BadMaths
+BadMaths -> NotesList: NotesList(Storage.loadFile(path))
+NotesList -> Storage : loadFile(String path)
+activate Storage
+deactivate Storage
+activate NotesList
+NotesList --> BadMaths
+deactivate NotesList
+BadMaths -> Parser : Parser(userInput)
+activate Parser
+Parser --> BadMaths : getCommand()
+Parser --> BadMaths : getToDo()
+deactivate Parser
+alt inputCommand is null
+activate Command
+    BadMaths -> Command : Command(command, toDo);
+    else inputCommand is not null
+    BadMaths -> Command : setCommand(command)
+    BadMaths -> Command : setToDo(toDo)
+    end
+BadMaths -> Command : executeCommand(notes)
+Command -> TrigoGraph : TrigoGraph(toDo)
+activate TrigoGraph
+deactivate TrigoGraph
+Command -> Calculator : Calculator(toDo)
+activate Calculator
+deactivate Calculator
+Command -> Quadratic : Quadratic(toDo)
+activate Quadratic
+deactivate Quadratic
+alt "Bye"
```

Comment:
Maybe you can phrase it more like a condition, e.g. `command is "Bye"`, so that it's clearer what this "Bye" is referring to